CRM/Contact/Form/Task/SaveSearch.php

```diff
@@ -208,12 +211,9 @@ public function postProcess() {
     $params = array();
     $params['title'] = $formValues['title'];
     $params['description'] = $formValues['description'];
-    if (isset($formValues['group_type']) &&
-      is_array($formValues['group_type'])
-    ) {
+    if (isset($formValues['group_type']) && is_array($formValues['group_type']) && count($formValues['group_type'])) {
```

Comment:
I think this is an unrelated change. It makes sense though from what I can see - if there is no group_type but it is set & is an array save as 'no group type'

Reply:
Yes, it fixes a PHP notice as the code was not checking for an empty array, but looking at it I believe it intended to as it doesn't make sense otherwise.  It's pulling a group type from the form into a variable, if there isn't one it can't do that so sets an empty string in the else below.